lib/evss/letters/letters_response.rb

```diff
@@ -19,7 +19,9 @@ def initialize(status, response = nil)
       end
 
       def metadata
-        super().merge(address: address)
+        meta = super
+        meta[:address] = address if ok?
```

Comment:
👍  it could move up into data, I'll circle back to it during our tech debt sprint